kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/checksum/Crc32ChecksumGeneratorTest.java

```diff
@@ -123,6 +111,58 @@ void shouldIncludeGenerationInChecksum() {
                 .isNotEqualTo(proxyChecksum);
     }
 
+    @Test
+    void shouldAppendGenerationToChecksum() {
+        // Given
+        String proxyChecksum = MetadataChecksumGenerator.checksumFor(List.of(PROXY));
```

Comment:
I don't understand this test.  MetadataChecksumGenerator.checksumFor uses a Crc32ChecksumGenerator itself.  Internally it calls Crc32ChecksumGenerator#appendMetadata.

So aren't the expected and actual paths computing the same thing using the same code?

Reply:
Hmm Yes, I was seeing some issues with appending metadata and was trying to figure out what was wrong. TLDR buffer positions within `appendLong`. So I was attempting to ensure that that the static method and the instance call produced the same result but your right there is not a lot of point in the test as it stands.

Reply:
I've removed the static `checksumFor` from the interface so I think this is resolved.